addons/product_expiry/wizard/confirm_expiry.py

```diff
@@ -36,7 +36,9 @@ def process(self):
         picking_to_validate = self.env.context.get('button_validate_picking_ids')
         if picking_to_validate:
             picking_to_validate = self.env['stock.picking'].browse(picking_to_validate).with_context(skip_expired=True)
```

Comment:
```suggestion
            picking_to_validate = self.env['stock.picking'].browse(picking_to_validate)
```

Reply:
Thanks for your suggestion, implemented with https://github.com/odoo/odoo/pull/76087/commits/fb7cbc037c80589fe565832e36171e08608c0ae8